channels/applicationinsights-channel-js/src/Interfaces.ts

```diff
@@ -180,6 +180,16 @@ export interface ISenderConfig {
      * @since 3.3.7
      */
     corsPolicy?: string;
+
+    /*
+     * [Optional] If set to false and the CompressionStream API is available, the payload will be compressed using the CompressionStream API.
+     * Compression will only occur if the event is asynchronous. For events like unloads, compression will not be applied.
+     * Note: if user set payloadPreprocessor, this zip compression will not be applied.
+     * @default true
+     * @since 4.3.7
+     * Starting from minor version 4.4, the default will be changed to false (compression enabled by default).
+    */
+    disableZip?: boolean;
```

Comment:
Please also create a 2nd follow-up task as we also want to add in the throttle manager task here to "enable" the compression after the fact. But we need to also ensure that we are remembering the previous cfg state.

Lets talk about this a little more during the team meeting (after creating the task)